tcms/xmlrpc/utils.py

```diff
@@ -30,14 +28,13 @@ def pre_check_product(values):
     else:
         product_str = values
 
-    if isinstance(product_str, str):
+    if type(product_str) == str:
```

Comment:
Why change ?

Reply:
My bad. The pylint check is the other way around. It issues a warning if we use `type(...)` instead of `isinstance(...)`. I have fixed that at both places it was used